package.json

```diff
@@ -7,6 +7,7 @@
   "workspaces": [
     "api",
     "contract",
+    "ui",
```

Comment:
can we exclude `/src/components/` so eslint doesn't have to look at jsx files?  Or is it an improvement to have those covered by eslint?

Reply:
Eslint has settings for JSX, so it's expected to lint those.